### PR TITLE
Adds Avali

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
Adds the Avali species and lots of clothing templates and clothings to fit them.
The Avali assets I've taken from Skyrat and modified and repurposed their "template" sprites.
I made a ton of sprites for the variants and templates aswell.

dreamseeker_2021-11-14_13-18-55
dreamseeker_2021-11-15_05-09-43

## Changelog
:cl:
add: Adds Avali
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/teshari.dm [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/datum/species/teshari //small voxes
```
Comment (atakiya) [inline code/modules/mob/living/carbon/human/species_types/teshari.dm:1]: what the fuck 😡

File: code/__DEFINES/mobs.dm [hunk 2]
```diff
@@ -290,7 +290,7 @@
 #define OFFSET_UNIFORM "uniform"
 #define OFFSET_ID "id"
 #define OFFSET_GLOVES "hands"
-#define OFFSET_GLASSES "glasses"
+#define OFFSET_GLASSES "eyes"
```
Comment (atakiya) [inline code/__DEFINES/mobs.dm:293]:
```suggestion
#define OFFSET_GLASSES "eyes"
```
Why the underlying enum change?

**Review by atakiya — COMMENTED** [review]
Will testmerge for a while.

Would really appreciate it if in the future, monolithic PRs like this would be split up in their sub parts.
The clothing addition for example could easily be separate and would make reviewing this much easier as its less to concentrate on at once.

File: code/__DEFINES/mobs.dm (hunk 2 above)
Comment (Azarak) [reply code/__DEFINES/mobs.dm:293]: The offset define is supposed to match the slot define. I suppose one could just define the offset defines with the slot defines.

Comment (atakiya) [reply code/__DEFINES/mobs.dm:293]: That makes sense, but yeah, stupid to have two defines for the same thing in that case.
Oh well.

**Comment by Azarak** [comment]
Yeah, sorry about that. I didn't want them to feel like an unfinished species when players get a hold of them. I usually try and atomize things, but I also often do big changes/refactors.

**Review by atakiya — APPROVED** [review]
TM has been in for a while, seems all good for a baseline update with patching done down the line.
Merging.

**Pull request merged**